sqlalchemy_aio/engine.py

```diff
@@ -293,12 +303,31 @@ def __init__(self, result_proxy, run_in_thread):
         self._result_proxy = result_proxy
         self._run_in_thread = run_in_thread
 
+    async def __aiter__(self):
```

Comment:
I had some trouble with async_generator.
I opted for a "traditional solution": a separate class with its own __aiter__ and __anext__.